anitya/lib/exceptions.py

```diff
@@ -117,7 +117,7 @@ class RateLimitException(AnityaException):
 
     def __init__(self, reset_time):
         self.reset_time = reset_time
-        self.dt_reset_time = dateutil.parser.parse(reset_time)
+        self.dt_reset_time = arrow.get(reset_time).naive
```

Comment:
I don't think it makes sense to throw away the timezone information here. I don't think we need to hold onto ``reset_time`` either. I'd recommend writing this as:

```
def __init__(self, reset_time):
    self.reset_time = arrow.get(reset_time)
```

This way you keep the timezone information, can accept anything arrow.get accepts, and there's not two attributes representing the same thing.

Reply:
I tried this and it didn't work, because the database isn't timezone aware and doesn't know how to store this information (at least in vagrant environment).